sql/core/src/test/scala/org/apache/spark/sql/execution/command/MsckRepairTableParserSuite.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.execution.command
+
+import org.apache.spark.sql.catalyst.analysis.{AnalysisTest, UnresolvedTable}
+import org.apache.spark.sql.catalyst.parser.CatalystSqlParser.parsePlan
+import org.apache.spark.sql.catalyst.plans.logical.RepairTable
+import org.apache.spark.sql.test.SharedSparkSession
+
+class MsckRepairTableParserSuite extends AnalysisTest with SharedSparkSession {
```

Comment:
- Is there a reason not to keep in `DDLParserSuite`?
- `SharedSparkSession` is required?

Reply:
> Is there a reason not to keep in DDLParserSuite of catalyst module?

Actually, there are two `DDLParserSuite`:
1. In `sql/core`: org.apache.spark.sql.execution.command.DDLParserSuite which checks parsing of DDL commands
2. In `sql/catalyst`: org.apache.spark.sql.catalyst.parser.DDLParserSuite which checks parsing of everything

Since the parser tests check parsing of particular DDL commands, I moved them closely to unified tests.

> SharedSparkSession is required?

No, it is not. Since it was copy-pasted from other *ParserSuite, I opened the JIRA SPARK-34447 to refactor other tests later (maybe improve other stuff), and to do not touch unrelated code in this PR.